Source/SwiftLintFramework/Rules/QuickDiscouragedCallRule.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  QuickDiscouragedCallRule.swift
+//  SwiftLint
+//
+//  Created by Ornithologist Coder on 8/11/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+// swiftlint:disable identifier_name
+
+public struct QuickDiscouragedCallRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "quick_discouraged_call",
+        name: "Quick Discouraged Call",
+        description: "Discouraged call inside 'describe' and/or 'context' block.",
+        kind: .style,
+        nonTriggeringExamples: QuickDiscouragedCallRuleExamples.nonTriggeringExamples,
+        triggeringExamples: QuickDiscouragedCallRuleExamples.triggeringExamples
+    )
+
+    public func validate(file: File,
+                         kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard
+            fileContainsQuickSpec(file: file),
```

Comment:
*in theory*, this could lead to false positives, as you could have one of that methods outside a `QuickSpec`. However, since this is an opt-in rule, I think we should be fine (until someone complains 😅)